src/Suggestor/Suggestor.ts

```diff
@@ -381,9 +381,11 @@ function addRecurrenceSuggestions(
             // We also add a nice checkmark in this case to denote it's a complete valid recurrence description
             const parsedRecurrence = Recurrence.fromText({
                 recurrenceRuleText: recurrenceString,
-                startDate: null,
-                scheduledDate: null,
-                dueDate: null,
+                occurrence: new Occurrence({
+                    startDate: null,
+                    scheduledDate: null,
+                    dueDate: null,
+                }),
```

Comment:
I don't feel super strongly about this, but I was envisaging that this new class would provide a little but of encapsulation of the dates, and reduce a little overhead around adding new  date values - of which `reminder` is one coming up at some point...

Is there anyway for the constructor to default all values to null, to avoid having to list all the fields?
Or could there be a static/named constructor maybe? `Occurrence.withNoDates()` or similar...